ydb/library/yql/sql/pg/pg_sql.cpp

```diff
@@ -4022,8 +4023,17 @@ class TConverter : public IPGParseEvents {
             return nullptr;
         }
 
-        if (value->sortby_nulls != SORTBY_NULLS_DEFAULT) {
-            AddError(TStringBuilder() << "sortby_nulls unsupported value: " << (int)value->sortby_nulls);
+        switch (value->sortby_nulls) {
+        case SORTBY_NULLS_DEFAULT:
+            break;
```

Comment:
Don't we want to preserve Postgres default behaviour here? "NULLS FIRST is the default for DESC order, and NULLS LAST otherwise."

Reply:
Nope. It's a breaking change